src/pages/style-guide/style-guide.jsx

```diff
@@ -64,85 +64,128 @@ var StyleGuide = React.createClass({
   render: function () {
     return (
       <div id="style-guide">
-        <Alert>Git Revision: {config.GIT_REVISION}</Alert>
-        <h1>Style Guide</h1>
-
-        <h2>Typography</h2>
-
-        <h1>Heading H1</h1>
-        <h2>Heading H2</h2>
-        <h3>Heading H3</h3>
-        <h4>Heading H4</h4>
-        <h5>Heading H5</h5>
-        <h6>Heading H6</h6>
-
-        <p>Paragraph: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris.</p>
-
-        <h2>Colors</h2>
-
-        <div className="swatches">
-          <div className="color blue"><span>@blue</span></div>
-          <div className="color shadowBlue"><span>@shadowBlue</span></div>
-          <div className="color sapphire"><span>@sapphire</span></div>
-          <div className="color teal"><span>@teal</span></div>
-          <div className="color slate"><span>@slate</span></div>
-          <div className="color heatherGrey"><span>@heatherGrey</span></div>
-          <div className="color lightGrey"><span>@lightGrey</span></div>
-          <div className="color softGrey"><span>@softGrey</span></div>
-          <div className="color plum"><span>@plum</span></div>
-          <div className="color shadowPlum"><span>@shadowPlum</span></div>
-          <div className="color brick"><span>@brick</span></div>
+        <div id="style-guide-container">
```

Comment:
Usually I avoid nesting IDs (really I generally try to use them as little as possible). 

In this case I'd use `class="wrapper"` with a `> .wrapper` selector under `#style-guide`.